lib/engines/content_block_manager/app/components/content_block_manager/content_block/document/show/document_timeline/timeline_item_component.html.erb

```diff
@@ -23,6 +23,13 @@
             version:,
             schema:,
             ) %>
+
+          <% embedded_object_diffs.each do |item| %>
```

Comment:
I think there needs to be a null check here, as seems to fail if there are no `embedded_object_diffs`

Reply:
That's a bit odd too, because it should always return at least an empty array? 🤔